marklogic-data-hub-central/ui/src/components/queries/saving/save-queries-dropdown/save-queries-dropdown.tsx

```diff
@@ -0,0 +1,84 @@
+import {Select} from "antd"
+import React, {useContext} from 'react';
+import styles from './save-queries-dropdown.module.scss';
+import { UserContext } from "../../../../util/user-context";
+import { SearchContext } from "../../../../util/search-context";
+import {fetchQueryById} from "../../../../api/queries";
+
+
+interface Props {
+    savedQueryList: any[];
+    toggleApply: (clicked:boolean) => void;
+    greyFacets: any[];
+};
+
+
+const SaveQueriesDropdown: React.FC<Props> = (props) => {
```

Comment:
I think the save query dropdown should be cleared whenever a search request is made. See examples.

https://drive.google.com/file/d/1lZWZGKOkRsbywOsd6R2W1A9H7JHAiWlH/view
https://drive.google.com/file/d/1BmWFT9ifZ45-ld3uit6sgoyTIs6Ip2Yy/view

Reply:
@brucean52 fixed this issue.